azure-pipelines.yml

```diff
@@ -10,8 +10,12 @@ stages:
     steps:
     - script: |
         sudo apt update
-        sudo apt install nuget
-        sudo apt install gcc-multilib
+        sudo apt install nuget gcc-multilib software-properties-common
+
+        sudo add-apt-repository ppa:deadsnakes/ppa
```

Comment:
Would it be possible to NOT use a PPA?

Reply:
cant python3.7 is not available in 18.04 lts

Reply:
We needed to use the UsePythonVersion task in Azure Pipelines